src/pefile.h

```diff
@@ -65,9 +61,38 @@ class PeFile : public Packer {
     virtual void setOhDataBase(const pe_section_t *osection) = 0;
     virtual void setOhHeaderSize(const pe_section_t *osection) = 0;
 
-    template <typename LEXX, typename ht>
-    void pack0(OutputFile *fo, ht &ih, ht &oh, unsigned subsystem_mask,
-               upx_uint64_t default_imagebase, bool last_section_rsrc_only);
+
```

Comment:
The explanations are a little more informative than the previous contents of the file.  But the lines longer than 80 to 100 characters are obnoxious, and should be broken into shorter lines.